.gitignore

```diff
@@ -23,3 +23,4 @@ server/www/img/icons/
 server/www/
 app/locales/
 package-lock.json
+app/stats.html
```

Comment:
curious, what's this?

Reply:
Something that Robbie added for testing.

Reply:
:D cool